.github/actions/directly-backup-and-restore-snapshot-database/action.yml

```diff
@@ -61,7 +65,22 @@ runs:
     - name: Backup database
       shell: bash
       run: |
-        bin/konduit.sh -n ${{ env.NAMESPACE }} cpd-ec2-${{ inputs.environment }}-web -- pg_dump -E utf8 --compress=1 --clean --if-exists --no-privileges --no-owner --verbose -f backup-${{ inputs.environment }}.sql.gz
+        # Build up exclude arguments based on flags
+        EXCLUDE_ARGS=""
+
+        if [ "${{ inputs.exclude-jobs }}" = "true" ]; then
+          EXCLUDE_ARGS="$EXCLUDE_ARGS \
```

Comment:
Have we got `pg_dump: error: too many command-line arguments` error in the [run without the tables](https://github.com/DFE-Digital/register-early-career-teachers-public/actions/runs/16937822324)? Why can't we add all solid queue tables instead?

Reply:
Ah I linked to the wrong run in the PR description; I've updated the link now. I hit that error at one point but it was because I was passing multiple tables to `--exclude-table-data` whereas you need to pass the option multiple times, once for each table.

We could exclude all solid queue tables but some of them contain information about scheduled/recurring jobs; it felt safer to keep these, though I suspect they would be re-created on boot.

Reply:
It'd be ok to remove all solid queue tables as we don't need scheduled or recurring jobs running in the new migration env either I reckon? But let's see how it goes.. happy to approve it 👍 tks